stable/home-assistant/templates/service.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "home-assistant.fullname" . }}
+  labels:
+    app: {{ template "home-assistant.name" . }}
+    chart: {{ template "home-assistant.chart" . }}
+    component: "{{ .Values.name }}"
```

Comment:
Your component names seem kind of inconsistent, e. g. `{{ .Release.Name }}-{{ .Values.name }}` vs. `{{ .Values.name }}`. Do component names have to be configurable at all? Why not hard-code them?

Reply:
I based this approach of configuring the component names based on how I observed the prometheus chart was written.  I used that concept as a framework for this one.  I'll hard-code the component names instead.

Reply:
One possible issue I see with hard-coding the component names is the `selector:` component name reference in the service type.  

If there happens to be another unique chart deployed sharing the same hard-coded component name (e.g. _server_), the service will 'select' the other same hard-coded-name component of the other chart.  It seems that the component names must be unique so that this naming collision condition cannot occur - unless I'm misunderstanding how this part works @unguiculus?

Reply:
I think I see now - additional things under the selector will prevent this and not cause an issue with common hard-coded component names